structure_tests/tests/structure_test_tests.sh

```diff
@@ -34,8 +34,7 @@ popd
 
 # Run the debian tests, they should always pass on latest
 envsubst < cloudbuild.yaml.in > cloudbuild.yaml
-gcloud container builds submit . --config=cloudbuild.yaml
-if [ "$?" -gt "0" ]
+if [ "$(gcloud container builds submit . --config=cloudbuild.yaml)" -gt "0" ]
```

Comment:
The return value is only used in the buildifier case, so that shellcheck should be disabled. The others I think are now correctly reformatted.